content/en/universal_service_monitoring/guide/using_usm_metrics.md

```diff
@@ -0,0 +1,121 @@
+
+You can access request, error, and duration metrics in Datadog for both inbound and outbound traffic on all services discovered with Universal Service Monitoring. These service health metrics that are useful for creating alerts, [tracking deployments][11], and getting started with [service level objectives (SLOs)][3] so you can get broad visibility into all services running on your infrastructure. 
+
+{{< img src="universal_service_monitoring/guide/usm_slo.png" alt="Universal Service Monitoring SLOs for BITSBOUTIQUE" style="width:100%;" >}}
+
+This guide describes how to search for USM metrics such as `universal.http.*` and use them in your monitors, SLOs, and dashboards.
+
+## USM metrics vs APM metrics
+
+| Metric Name                 | Units   | Type         | Description                                       |
+|-----------------------------|---------|--------------|---------------------------------------------------|
+| universal.http.client       | Seconds | Distribution | Total number of outbound requests.                |
+| universal.http.client.apdex |         | Gauge        | Total number of outbound errors.                  |
+| universal.http.server       | Seconds | Distribution | Total number of outbound requests by HTTP status. |
+| universal.http.server.apdex |         | Gauge        | The Apdex score for each web service.             |
+
+### Metric syntax
+
+The USM metric query syntax differs from the [APM metric query syntax][4], which uses `trace.*`. USM Metrics fall under a single distribution metric name. 
+
+For example:
+
+| APM                                             | USM                                                  |
+|-------------------------------------------------|------------------------------------------------------|
+| trace.universal.http.client.hits{*}             | count:universal.http.client{*}                       |
+| trace.universal.http.client.errors              | count:universal.http.client{error:true}              |
+| trace.universal.http.client.hits.by_http_status | count:universal.http.client{*} by http_status_family |
+| pXX:trace.universal.http.client{*}              | pXX:universal.http.client{*}                         |
+| trace.universal.http.client.apdex{*}            | universal.http.client.apdex{*}                       |
+
+The same translations apply for the `universal.http.server` operation that captures inbound traffic. For more information about distribution metrics, see [DDSketch-based Metrics in APM][12].
+
+## Usage
+
+Navigate to [**APM** > **Service Catalog**][5] and click on a service. The **Performance** tab displays service-level graphs on hits, latency, requests, errors, and more. You can also access these metrics when creating a [monitor](#create-a-monitor) or an [SLO](#create-an-slo), or by looking at a [dashboard](#access-a-defined-dashboard) in the [Service Catalog][2].
```

Comment:
Do we need to specify here that they should look at USM-identified services? Something like:

`Navigate to [**APM** > **Service Catalog**][5], filter by Universal Service Monitoring telemetry type, and click on a service. `

Reply:
```suggestion
Navigate to [**APM** > **Service Catalog**][5], filter by a Universal Service Monitoring telemetry type, and click on a service. The **Performance** tab displays service-level graphs on hits, latency, requests, errors, and more. You can also access these metrics when creating a [monitor](#create-a-monitor) or an [SLO](#create-an-slo), or by looking at a [dashboard](#access-a-defined-dashboard) in the [Service Catalog][2].
```